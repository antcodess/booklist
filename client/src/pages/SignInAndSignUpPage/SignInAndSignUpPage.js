import React from 'react';

import BreadCrumbNav from '../../components/BreadCrumbNav/BreadCrumbNav';
import SignIn from '../../components/SignIn/SignIn';
import SignUp from '../../components/SignUp/SignUp';

import './SignInAndSignUpPage.scss';

const SignInAndSignUpPage = () => {
  return (
    <React.Fragment>
      <BreadCrumbNav title='SignIn & SignUp' />
      <div className='sign-in-and-sign-up'>
        <SignIn />
        <SignUp />
      </div>
    </React.Fragment>
  );
};

export default SignInAndSignUpPage;