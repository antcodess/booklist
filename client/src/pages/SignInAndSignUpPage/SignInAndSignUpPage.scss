.sign-in-and-sign-up {
  display: flex;
  justify-content: space-between;
  
  h2 {
    text-align: left;
    margin-top: 20px;
    font-size: 2rem; 
  }
}

@media only screen and (max-width: 720px) {
  .sign-in-and-sign-up {
    flex-direction: column;
    justify-content: center;
  }
}