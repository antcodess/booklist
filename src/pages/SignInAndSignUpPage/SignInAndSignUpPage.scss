.sign-in-and-sign-up {
  display: flex;
  justify-content: space-between;
  
  h2 {
    text-align: left;
    margin-top: 20px;
    font-size: 2rem; 
  }
}