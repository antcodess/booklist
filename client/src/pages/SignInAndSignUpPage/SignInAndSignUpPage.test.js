import React from 'react';
import { shallow } from 'enzyme';
import { SignInAndSignUpPage } from './SignInAndSignUpPage';

describe('SignInAndSignUpPage Component', () => {
  let wrapper;
  
  beforeEach(() => {
    wrapper = shallow(<SignInAndSignUpPage />);
  });

  it('should render SignInAndSignUpPage Component', () => {
    expect(wrapper).toMatchSnapshot();
  });
});