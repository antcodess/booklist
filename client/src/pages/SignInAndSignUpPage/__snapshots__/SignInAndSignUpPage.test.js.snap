// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignInAndSignUpPage Component should render SignInAndSignUpPage Component 1`] = `ShallowWrapper {}`;
